gratipay/models/team/mixins/takes.py

```diff
@@ -3,6 +3,7 @@
 from collections import OrderedDict
 from decimal import Decimal as D
 
+from gettext import gettext as _
```

Comment:
This question goes away if we go with preventing adding in the first place as a solution to the UX problem with adding invalid users (https://github.com/gratipay/gratipay.com/pull/4077#discussion_r73422575).